File: sample-java/src/main/java/social/bigbone/sample/GetInstanceInfo.java
package social.bigbone.sample;

import social.bigbone.MastodonClient;
import social.bigbone.api.entity.Instance;
import social.bigbone.api.exception.BigBoneRequestException;

@SuppressWarnings("PMD.SystemPrintln")
public class GetInstanceInfo {

    public static void main(final String[] args) throws BigBoneRequestException {
        final String instance = args[0];

        // Instantiate client
        final MastodonClient client = new MastodonClient.Builder(instance).build();

        // Get instance info and dump it to the console as JSON
        final Instance instanceInfo = client.instances().getInstance().execute();

        System.out.println(instanceInfo.toString());
    }
}

File: sample-kotlin/src/main/kotlin/social/bigbone/sample/GetInstanceInfo.kt
package social.bigbone.sample

import social.bigbone.MastodonClient

object GetInstanceInfo {
    @JvmStatic
    fun main(args: Array<String>) {
        val instance = args[0]

        // Instantiate client
        val client = MastodonClient.Builder(instance)
            .build()

        // Get instance info and dump it to the console as JSON
        val instanceInfo = client.instances.getInstance().execute()
        println(instanceInfo.toString())
    }
}
